src/pages/account/logins/[type]/new.tsx

```diff
@@ -1,14 +1,62 @@
+import { HttpStatus } from "@fewlines/fwl-web";
+import {
+  JsonWebTokenError,
+  NotBeforeError,
+  TokenExpiredError,
+} from "jsonwebtoken";
 import { GetServerSideProps } from "next";
 import React from "react";
 
-const NewIdentity: React.FC = () => {
-  return <React.Fragment />;
+import Sentry, { addRequestScopeToSentry } from "../../../..//utils/sentry";
+import { IdentityTypes } from "../../../../@types/Identity";
+import { AddIdentity } from "../../../../components/business/AddIdentity";
+import { AddIdentityInputForm } from "../../../../components/display/fewlines/AddIdentityInputForm";
+import { withSSRLogger } from "../../../../middleware/withSSRLogger";
+import withSession from "../../../../middleware/withSession";
+
+const NewIdentity: React.FC<{ type: IdentityTypes }> = (props) => {
+  return (
+    <AddIdentity type={props.type}>
+      {({ addIdentity }) => (
+        <AddIdentityInputForm addIdentity={addIdentity} type={props.type} />
+      )}
+    </AddIdentity>
+  );
 };
 
 export default NewIdentity;
 
-export const getServerSideProps: GetServerSideProps = async (_context) => {
-  return {
-    props: {},
-  };
-};
+export const getServerSideProps: GetServerSideProps = withSSRLogger(
+  withSession(async (context) => {
+    addRequestScopeToSentry(context.req);
+    try {
+      return {
+        props: {
+          type: context.params.type,
+        },
+      };
+    } catch (error) {
+      if (
+        error instanceof JsonWebTokenError ||
+        error instanceof NotBeforeError ||
+        error instanceof TokenExpiredError
+      ) {
+        Sentry.withScope((scope) => {
+          scope.setTag(
+            "/pages/account/logins SSR",
+            "/pages/account/logins SSR",
```

Comment:
You need to change the scope tags to reflect where the error will happen.

Reply:
OK fixed! a367648

Reply:
User id is sensitive data, we don't want to monitor them.

Reply:
To extends my comment (as I was in the rush on the other repo).

It's ok to have the user ID in query strings, but we don't want to save it on our side (which will be the case if you add it in the trace, because we store them).

Reply:
I only was using user ID in `src/pages/account/logins/[type]/[id]/[action].tsx`, but this file has been deleted afterwards 0ccaa8e